packages/big-design/src/components/ProgressCircle/ProgressCircle.tsx

```diff
@@ -58,11 +44,9 @@ export class ProgressCircle extends React.PureComponent<ProgressCircleProps> {
         )}
       </StyledProgressCircle>
     );
-  }
-
-  renderError() {
-    const { percent } = this.props;
+  };
 
+  const renderError = () => {
```

Comment:
🍸 I think you could return ErrorIcon immediately. `const renderError = () => (...);`